tests/testthat/test-submit-model.R

```diff
@@ -111,3 +114,46 @@ test_that("submit_model(.mode) errors when invalid", {
     regexp = "Invalid value passed.+mode"
   )
 })
+
+test_that("submit_model aborts if .mode='sge' is used with Slurm's qsub", {
+  skip_if_old_bbi("3.4.0")
+
+  sbatch <- unname(Sys.which("sbatch"))
+  if (identical(sbatch, "") || identical(Sys.getenv("METWORX_VERSION"), "")) {
+    skip("not on Metworx with SGE")
```

Comment:
Is this supposed to be `"not on Metworx with Slurm"` (instead of `SGE`)?

It seems like you would hit this when `sbatch` doesn't show up, and the point of this test is for systems _with_ Slurm.

Reply:
Doh, thanks for catching that.

Reply:
I've amended the commit to say Slurm instead of SGE ([diff](https://github.com/metrumresearchgroup/bbr/compare/25a7f1c5dbc9371457c1960939d743c4540ab997..01f39e6fd4a7bbc6dfc419336e83053a1ba7dfad)).